params/config.go

```diff
@@ -36,14 +36,14 @@ var MainnetChainConfig = &ChainConfig{
 
 // TestnetChainConfig is the chain parameters to run a node on the test network.
 var TestnetChainConfig = &ChainConfig{
-	ChainId:        TestNetChainID,
-	HomesteadBlock: TestNetHomesteadBlock,
-	DAOForkBlock:   TestNetDAOForkBlock,
-	DAOForkSupport: false,
-	EIP150Block:    TestNetHomesteadGasRepriceBlock,
-	EIP150Hash:     TestNetHomesteadGasRepriceHash,
-	EIP155Block:    TestNetSpuriousDragon,
-	EIP158Block:    TestNetSpuriousDragon,
+	ChainId:        big.NewInt(3),
+	HomesteadBlock: big.NewInt(0),
+	DAOForkBlock:   nil,
+	DAOForkSupport: true,
+	EIP150Block:    big.NewInt(0),
+	EIP150Hash:     common.HexToHash("0x41941023680923e0fe4d74a34bdac8141f2540e3ae90623718e47d66d1ca4a2d"),
+	EIP155Block:    big.NewInt(10),
+	EIP158Block:    big.NewInt(10),
 }
```

Comment:
I want to move away from defining the variables in another file. I'd like to move to using `ChainConfig` instead of having to set fields to a new `ChainConfig`, which is what we do currently in the flags.go file.